docs/source/providers/vector_io/sqlite-vec.md

```diff
@@ -10,11 +10,41 @@ That means you're not limited to storing vectors in memory or in a separate serv
 ## Features
 
 - Lightweight and easy to use
-- Fully integrated with Llama Stack
+- Fully integrated with Llama Stacks
+- Uses disk-based storage for persistence, allowing for larger vector storage
+
+### Comparison to Faiss
+
+SQLite-Vec is a lightweight alternative to Faiss, which is a popular vector database provider.
+While Faiss is a fast, lightweight and powerful inline provider, Faiss reindexes the
+entire database when a new vector is added. SQLite-Vec is a disk-based storage provider
+that allows for larger vector storage and handles incremental writes more efficiently.
+
+SQLite-vec is a great alternative to Faiss when you need to execute several writes to the
```

Comment:
@leseb this is where I tried to clarify the trade off.

Reply:
I think mentioning the OLTP and OLAP use cases would help users better than just saying "execute several writes" :).

Reply:
I updated and added that to the table. I also removed this wording and tried to just restate the data to be a bit more neutral.